geomesa-hbase/geomesa-hbase-datastore/src/main/scala/org/locationtech/geomesa/hbase/data/HBaseDataStoreFactory.scala

```diff
@@ -83,6 +87,7 @@ object HBaseDataStoreFactory {
   object Params {
     val BigTableNameParam  = new Param("bigtable.table.name", classOf[String], "Table name", true)
     val ConnectionParam    = new Param("connection", classOf[Connection], "Connection", false)
+    val RemoteParam        = new Param("remote filtering", classOf[Boolean], "Remote filtering", false)
```

Comment:
yeah, the shared parameters among data stores follow the camel-case convention, so most of them follow that pattern... maybe we should deprecate those and move to dot convention...